config/job.js

```diff
@@ -77,34 +77,61 @@ const SCHEMA_ENVIRONMENT = Joi.object()
     + 'and underscore (cannot start with digit)'
     });
 const SCHEMA_JOBNAME = Joi.string().max(100).regex(Regex.JOB_NAME);
+// Step can be in the following formats:
+// npm install
+// { init: npm install }
+// { init: { command: npm install } }
 const SCHEMA_STEP_STRING = Joi.string();
+const SCHEMA_STEP_SUBOBJECT_BASE = Joi.object()
+    .keys({
+        command: SCHEMA_STEP_STRING.required()
```

Comment:
does job step need to have locked to determine that it's not overridable ? Or will we be looking into template step  config for this info ?

Reply:
this will only be used for validation

but job step doesn't need `locked`
removed